.github/workflows/ci.yml

```diff
@@ -7,10 +7,35 @@ on:
   pull_request:
 
 jobs:
+  run-hello-world:
+    strategy:
+      matrix:
+        os: [windows-latest]
+
+    name: Build Effekt compiler and runs one program
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: 'true'
+
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+
+    - name: Set up NodeJS
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+
+    - name: Run tests
+      run: sbt "effektJVM/clean; effektJVM/run examples/pos/sideeffects.effekt"
+
   build-jar:
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latests]
```

Comment:
This might be a typo ;) 
(the tests are waiting for almost 2 hours for a machine of this name -- I'll just cancel those)

Reply:
It is. ****.
Thanks for cancelling!